### PR TITLE
Merge Static Components

### PR DESCRIPTION
**Pull request by GuyInFridge**
- Fixed scalings on various webpages, notably those with graphics on them
- Twiddled around with the static pages, including About / FAQ / Contact us, making them neater and more up to date
- There may be other random tweaks made if I felt that I was modifying a page that was 'static' enough .-.

### CODE REVIEW

File: src/app/pages/about/about.component.html [hunk 1]
```diff
@@ -12,9 +12,9 @@ <h1 class="mb-3" id="title">About Us</h1>
         pandemic leading to lockdowns and social distancing, we wanted to create an alternative way for graduating
         students (and
         others!) to be able to send each other messages in a medium that they can cherish with them as they continue on
-        in their journeys.
+        in their journeys. <br><br>
+        Want to get in touch with us? Visit the <a routerLink="/contact">Contact Us</a> page.
       </p>
```
Comment (ricetech) [inline src/app/pages/about/about.component.html:17]:
```suggestion
        in their journeys.
      </p>
      <p>Want to get in touch with us? Visit the <a routerLink="/contact">Contact Us</a> page.</p>
```

File: src/app/pages/contact/contact.component.html [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+        <strong>Got a question, comment, suggestion or concern?</strong><br>
+        <a href="https://github.com/virtrolio/virtrolio-site/issues/new" target="_blank">Open an issue on
+        our GitHub</a> or email us at
+        <a href="mailto:virtrolio.team@gmail.com" target="_blank">virtrolio.team@gmail.com</a>.
       </p>
       <p>
-        You can also get in touch with us through our <a href="https://www.facebook.com/virtrolio/" target="_blank">
-        Facebook</a> or <a href="https://www.instagram.com/virtrolio/" target="_blank">Instagram</a> pages.
+        <strong>Looking for our social media?</strong><br>
+        You're welcome to visit our <a href="https://www.facebook.com/virtrolio/" target="_blank">
```
Comment (ricetech) [inline src/app/pages/contact/contact.component.html:14]:
```suggestion
        You're welcome to visit and contact us through our <a href="https://www.facebook.com/virtrolio/" target="_blank">
```

File: src/app/pages/faq/faq.component.html [hunk 3]
```diff
@@ -54,17 +54,23 @@ <h2 class="mb-0">
...
+          Yes and no. For any one version of the website (for example, if you just created an account with us), then you
+          will only be able to send one message to each of your friends. You probably wouldn't
+          sign someone's yearbook more than once in real life
+          (unless
           you just <em>wanted</em> to take space away from others). As such, we decided that you can send one very
-          special, meaningful message to each of your friends. Make it count!
+          special, meaningful message. However, if we make a major update to the website,
+          then we will allow for <strong>one extra message to be sent to each of your friends</strong> so that you can
+          try out the new update's
+          features.
```
Comment (ricetech) [inline src/app/pages/faq/faq.component.html:73]: Unnecessary line break
```suggestion
          try out the new update's features.
```

File: src/app/pages/faq/faq.component.html [hunk 4]
```diff
@@ -83,7 +89,7 @@ <h2 class="mb-0">
       <!-- Answer -->
       <div id="s1q4c" class="collapse" aria-labelledby="s1q4h" data-parent="#faq-s1">
         <div class='card-body card-align'>
-          Go to <a routerLink="/your-virtrolio">your Virtrolio</a> and click on 'Sharing Link,', then generate a new
+          Go to <a routerLink="/your-virtrolio">your Virtrolio</a> and click on 'Sharing Link,' then generate a new
```
Comment (ricetech) [inline src/app/pages/faq/faq.component.html:92]: You removed the wrong comma.
```suggestion
          Go to <a routerLink="/your-virtrolio">your Virtrolio</a> and click on 'Sharing Link', then generate a new
```

File: src/app/pages/faq/faq.component.html [hunk 5]
```diff
@@ -103,9 +109,33 @@ <h2 class="mb-0">
       <!-- Answer -->
       <div id="s1q5c" class="collapse" aria-labelledby="s1q5h" data-parent="#faq-s1">
         <div class="card-body card-align">
-          No. To be able to store images and doodles on our service, we would require funds to cover storage costs. We are looking at
-          ways to embed images that are stored elsewhere online. For now, feel free to send
-          your friends links to online photos or photo albums!
+          If you know or are willing to learn a little bit of HTML, then yes! You can add an image tag like this:
```
Comment (ricetech) [inline src/app/pages/faq/faq.component.html:112]: Friendlier wording
```suggestion
          We're working on adding a button to allow you to do this soon!
          However, if you know a bit of HTML, you can do it yourself like this:
```

File: src/app/pages/faq/faq.component.html [hunk 6]
```diff
@@ -124,7 +154,30 @@ <h2 class="mb-0">
...
+            Why would I want a new sharing link?
+          </button>
+        </h2>
+      </div>
+
+      <!-- Answer -->
+      <div id="s1q7c" class="collapse" aria-labelledby="s1q7h" data-parent="#faq-s1">
+        <div class="card-body card-align">
+          You can generate a new sharing link if people are sending you messages you do not approve of using your old
+          one. This helps prevent spammers and bullies from clogging up your virtrolio.
```
Comment (ricetech) [inline src/app/pages/faq/faq.component.html:180]:
```suggestion
          one. This helps prevent spammers and bullies from clogging up your virtrolio.
          Only people with your <em>newest</em> sharing link can sign your virtrolio.
```

File: src/app/pages/page-not-found/page-not-found.component.css [hunk 7]
```diff
@@ -1,12 +1,28 @@
-#pnf-graphic {
-  height: 50vh;
+#graphic {
```
Comment (ricetech) [inline src/app/pages/page-not-found/page-not-found.component.css:1]:
```suggestion
#pnf-graphic {
```

File: src/app/pages/page-not-found/page-not-found.component.html [hunk 8]
```diff
@@ -11,9 +11,9 @@ <h1 class="display-4" id="title-text">The page you are looking for doesn't exist
         href="mailto:virtrolio.team@gmail.com">virtrolio.team@gmail.com</a></p>
   </div>
   <div class="row justify-content-center mx-3 mt-3">
-    <img src="../../../assets/images/PageNotFound.svg" alt="Page Not Found graphic" id="pnf-graphic">
+    <img src="../../../assets/images/PageNotFound.svg" alt="Page Not Found graphic" id="graphic">
```
Comment (ricetech) [inline src/app/pages/page-not-found/page-not-found.component.html:14]:
```suggestion
    <img src="../../../assets/images/PageNotFound.svg" alt="Page Not Found graphic" id="pnf-graphic">
```

File: src/app/pages/about/about.component.html [hunk 9]
```diff
@@ -184,6 +184,36 @@ <h1 class="mb-4" id="meet-the-team-title">Meet the Team!</h1>
...
+          piano and travel (RIP). He also writes about math things, lands planes, and has made it out of REV alive.</p>
+      </div>
+
+      <!-- Anthony's Contact Info. -->
+      <div class="col-md-3 order-md-1">
+        <p><strong>Contact Anthony:</strong></p>
+        <ul>
+          <li>LinkedIn: <a href="linkedin.com/in/anthonyftwang" target="_blank">Anthony Wang</a></li>
+          <li>Github: <a href="github.com/anthonyftwang" target="_blank">Anthonyftwang</a></li>
+          <li>Instagram: <a href="instagram.com/itsanthonywang" target="_blank">@itsanthonywang</a></li>
```
Comment (ricetech) [inline src/app/pages/about/about.component.html:210]: Link corrections
```suggestion
          <li>LinkedIn: <a href="https://www.linkedin.com/in/anthonyftwang" target="_blank">Anthony Wang</a></li>
          <li>Github: <a href="https://github.com/anthonyftwang" target="_blank">Anthonyftwang</a></li>
          <li>Instagram: <a href="https://www.instagram.com/itsanthonywang" target="_blank">@itsanthonywang</a></li>
```

File: src/app/pages/about/about.component.html [hunk 10]
```diff
@@ -12,9 +12,9 @@ <h1 class="mb-3" id="title">About Us</h1>
...
         students (and
         others!) to be able to send each other messages in a medium that they can cherish with them as they continue on
-        in their journeys.
+        in their journeys. <br><br>
+        Want to get in touch with us? Visit the <a routerLink="/contact">Contact Us</a> page.
       </p>
-      <p>Want to get in touch with us? Visit the <a routerLink="/contact">Contact Us</a> page.</p>
     </div>
   </div>
 </div>
```
Comment (janakitti) [inline src/app/pages/about/about.component.html]: Add padding to the bottom to separate content from the footer a bit more.
I'm not sure why Github isn't letting me add a suggestion to this particular line (line 23)

Change
`<div class="mb-3" id="meet-the-team">`

to
`<div class="mb-5" id="meet-the-team">`

File: src/app/pages/about/about.component.html [hunk 11]
```diff
@@ -184,6 +184,36 @@ <h1 class="mb-4" id="meet-the-team-title">Meet the Team!</h1>
     </div>
     <hr/>
 
+    <!-- Anthony's Row-->
+    <div class="row d-flex">
+
+      <!-- Anthony's Image -->
+      <div class="col-md-2 order-md-2 image-container">
+        <img alt="Picture of Anthony Zhou" src="{{ Anthony.regularLink }}" class="img-fluid regular-picture">
+        <img alt="Silly picture of Anthony Zhou" src="{{ Anthony.sillyLink }}" class="img-fluid silly-picture">
```
Comment (janakitti) [inline src/app/pages/about/about.component.html:193]:
```suggestion
        <img alt="Picture of Anthony Wang" src="{{ Anthony.regularLink }}" class="img-fluid regular-picture">
        <img alt="Silly picture of Anthony Wang" src="{{ Anthony.sillyLink }}" class="img-fluid silly-picture">
```

Comment (ricetech) [reply src/app/pages/about/about.component.html:193]: What do you mean? He's CLEARLY Anthony Zhou!

File: src/app/pages/about/about.component.html [hunk 12]
```diff
@@ -184,6 +184,36 @@ <h1 class="mb-4" id="meet-the-team-title">Meet the Team!</h1>
...
+          currently studying Software Engineering at the University of Waterloo. In his spare time, he likes to play the
+          piano and travel (RIP). He also writes about math things, lands planes, and has made it out of REV alive.</p>
+      </div>
+
+      <!-- Anthony's Contact Info. -->
+      <div class="col-md-3 order-md-1">
+        <p><strong>Contact Anthony:</strong></p>
+        <ul>
+          <li>LinkedIn: <a href="https://www.linkedin.com/in/anthonyftwang" target="_blank">Anthony Wang</a></li>
+          <li>Github: <a href="https://www.github.com/anthonyftwang" target="_blank">Anthonyftwang</a></li>
```
Comment (ricetech) [inline src/app/pages/about/about.component.html:209]: GitHub doesn't use www
```suggestion
          <li>Github: <a href="https://github.com/anthonyftwang" target="_blank">Anthonyftwang</a></li>
```

**Pull request merged**